src/main/kotlin/no/nav/syfo/Metrics.kt

```diff
@@ -23,6 +24,12 @@ val COUNT_ENDRE_PERSON_STATUS_FORBIDDEN: Counter = Counter.build()
     .help("Counts the number of forbidden flaggings")
     .register()
 
+val COUNT_GET_PERSON_STATUS_FAIL: Counter = Counter.build()
+    .namespace(METRICS_NS)
+    .name(GET_PERSON_STATUS_FAIL)
+    .help("Counts the number of forbidden status checks")
+    .register()
```

Comment:
`Counts the number of forbidden status checks` Kan `get status` kun feile hvis handlingen er forbudt?